initialize-bitcoin.sh

```diff
@@ -13,4 +13,8 @@ printf "${LOG_START}Installing bitcoind-wallet...${LOG_END}"
 
 npm install -g "$WORKDIR/bitcoind-wallet"
 
+printf "${LOG_START}Cleaning up data directory...${LOG_END}"
+
+rm -rf $PWD/bitcoin/.data
```

Comment:
For consistency with the line 14 we could do:

```suggestion
rm -rf $WORKDIR/bitcoin/.data
```